tests/test_env.py

```diff
@@ -341,9 +284,16 @@ def test_active_env_get_module_file(
     if package_name.startswith(module.ModuleV2.PKG_NAME_PREFIX):
         module_name = "inmanta_plugins." + package_name[len(module.ModuleV2.PKG_NAME_PREFIX) :].replace("-", "_")
         index = str(local_module_package_index)
+        pip_config = PipConfig(
+            index_url=index,
+            use_system_config=True,  # we need an upstream for some packages
```

Comment:
You set both an index and the system config. Is this a mistake?

Reply:
seems to be not required.

Reply:
So the venv itself is isolated, as intended 

> Activates the venv created by the `tmpvenv` fixture within the currently running process. This venv is completely decoupled  from the active development venv.

But `inmanta-module-minimalv2module` still depends on core (which it somehow can get) but it can't get the dependencies of core.


I don't know how it worked before.

Reply:
@sanderr I have no clue how to handle this.

Reply:
same issue on test_active_env_get_module_file_editable_namespace_package